app/wedding/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from wedding import views
+
+router = DefaultRouter()
+router.register('weddings', views.WeddingViewSet)
+
+app_name = 'wedding'
+
+urlpatterns = [
+    path('', include(router.urls))
```

Comment:
Will we be adding a path later? Or is it just the app/url.py file that we'll be updating?